docker/test/performance-comparison/compare.sh

```diff
@@ -61,7 +61,7 @@ function configure
     cp -rv right/config left ||:
 
     # Start a temporary server to rename the tables
-    while pkill clickhouse-serv; do echo . ; sleep 1 ; done
+    while pkill -f clickhouse-serv ; do echo . ; sleep 1 ; done
```

Comment:
Interesting, why this is required? `clickhouse-serv` should be enough without full pattern match. Maybe it is because of watchdog? But it should not be since it does not bind and cannot inherit that fd too.

Reply:
idk, tbh

Reply:
Was it really servers that was still alive that was acquired the port?

Reply:
yes, as I described in the pr description